api_proxy.yaml

```diff
@@ -393,6 +416,22 @@ definitions:
       - name
       - preferred_email
       - version
+  PublicSession:
+    type: object
+    title: User session data
+    description: Describe the current session of an authenticated user.
+    properties:
+      expired:
+        type: boolean
+      expireAt:
+        type: number
+      newToken:
+        type: string
+      spidLevel:
```

Comment:
this should probably be an enum

Reply:
ref. https://github.com/teamdigitale/digital-citizenship-functions/blob/master/api/definitions.yaml#L37

Reply:
The problem here is that the possible keys are in the format: `https://www.spid.gov.it/SpidL1` so the generated enum will be:

```
export enum SpidLevelEnum {
  "https://www.spid.gov.it/SpidL1" = "https://www.spid.gov.it/SpidL1",

  "https://www.spid.gov.it/SpidL2" = "https://www.spid.gov.it/SpidL2",

  "https://www.spid.gov.it/SpidL3" = "https://www.spid.gov.it/SpidL3"
}
```

and `SpidLevelEnum.https://www.spid.gov.it/SpidL1` is not valid. Or there is a different way to reference those enums?

Reply:
SpidLevelEnum["https://www.spid.gov.it/SpidL1"]